pkg/api/pullrequestpayloadqualification/v1/types.go

```diff
@@ -123,3 +124,9 @@ type PullRequestPayloadQualificationRunList struct {
 
 	Items []PullRequestPayloadQualificationRun `json:"items"`
 }
+
+// JobName maps the name in the spec to the corresponding Prow job name.
+// It matches the `ReleaseJobName` value in the status.
+func (s ReleaseJobSpec) JobName() string {
+	return s.CIOperatorConfig.SimpleJobName(jobconfig.PeriodicPrefix, s.Test)
```

Comment:
Don't we need `JobName`?

(why do we enven have `SimpleJobName` / `JobName`?

Reply:
You're right, that was based on a previous (erroneous) idea about what the job names were, I didn't notice when I reinstated it.  Corrected.

(I also found the number of random combinations of `Metadata -> string` functions amusing)